crates/relayer-cli/src/commands/listen.rs

```diff
@@ -161,7 +161,9 @@ fn subscribe(
                     interval,
                     max_retries,
                 } => {
-                    let mut rpc_client = HttpClient::new(config.rpc_addr.clone())
+                    let mut rpc_client = HttpClient::builder(config.rpc_addr.clone().try_into()?)
+                        .user_agent(format!("hermes/{}", ibc_relayer::HERMES_VERSION))
```

Comment:
Let's import the constant instead of using its fully qualified name

```suggestion
                        .user_agent(format!("hermes/{}", HERMES_VERSION))
```